packages/xo-web/src/common/custom-fields.js

```diff
@@ -47,19 +74,57 @@ const AddCustomFieldModal = decorate([
           />
         </Col>
       </SingleLineRow>
-      <SingleLineRow className='mt-1'>
-        <Col size={6}>{_('value')}</Col>
-        <Col size={6}>
-          <input
-            className='form-control'
-            name='value'
-            onChange={effects.onChange}
-            required
-            type='text'
-            value={value.value}
-          />
-        </Col>
-      </SingleLineRow>
+      {value.isDate ? (
+        [
+          <SingleLineRow className='mt-1' key='date'>
+            <Col size={6}>
+              {_('date')} {_('utc')}
```

Comment:
The order may not make sense in some languages. You can create a "UTC date" message if you need to.